Paranormal/Paranormal/ToolSettings.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+public class ToolSettings {
+    weak var document : Document?
+
+    init(){
+    }
+
+    public var colorAsAngles : Array<Float> = [0,0] {
+        didSet {
+            updateToolSettingsView()
+        }
+    }
+
+    public func updateToolSettingsView() {
+        let panelVC = document?.singleWindowController?.panelsViewController
+        panelVC?.toolSettingsViewController?.updateSettingsSliders()
+    }
+
+    public var colorAsNSColor : NSColor {
+        let dir = colorAsAngles[0] * Float(M_PI) / 180.0
```

Comment:
Instead of weirdness like this you could just do colorAsAngles.dirAsAngle or something.